components/map.jsx

```diff
@@ -110,6 +110,7 @@ var MarkerPopupClub = React.createClass({
         <br/>
         <br/>
         <p>{this.props.description}</p>
+        <p><small>Led by <a href={"https://webmaker.org/en-US/search?type=user&q=" + this.props.owner}>{this.props.owner}</a></small></p>
```

Comment:
I mused about this for a bit.

Ultimately, I considered a club-view component. however, I came to the conclusion that the current club list is not scalable and needs to be updated somehow anyway, that doing too much on the engineering end of things might be not all that useful.

I feel like a component just for the link is too much, and I'm not sure how to make it a function a bridge the map to club page file gap.

I say we punt these problems.